modules/breakpad/SCsub

```diff
@@ -0,0 +1,196 @@
+        "src/client/windows/crash_generation/crash_generation_client.cc",
+        "src/client/windows/crash_generation/crash_generation_server.cc",
+        "src/client/windows/crash_generation/minidump_generator.cc",
+        "src/client/windows/handler/exception_handler.cc",
+        # "src/common/windows/dia_util.cc",
+        "src/common/windows/guid_string.cc",
+        # "src/common/windows/omap.cc",
+        # "src/common/windows/pdb_source_line_writer.cc",
+        "src/common/windows/pe_source_line_writer.cc",
+        # "src/common/windows/pe_util.cc",
+        "src/common/windows/string_utils.cc",
+    ]
+
+if env["platform"] == "osx":
+    breakpad_src += [
+        "src/client/mac/Breakpad.xcodeproj/project.pbxproj",
+        "src/client/mac/Framework/Breakpad.mm",
+        "src/client/mac/Framework/Breakpad_Prefix.pch",
+        "src/client/mac/Framework/Info.plist",
+        "src/client/mac/Framework/OnDemandServer.mm",
+        "src/client/mac/UnitTests-Info.plist",
+        "src/client/mac/crash_generation/ConfigFile.mm",
+        "src/client/mac/crash_generation/Inspector.mm",
+        "src/client/mac/crash_generation/InspectorMain.mm",
+        "src/client/mac/crash_generation/crash_generation_client.cc",
+        "src/client/mac/crash_generation/crash_generation_server.cc",
+        "src/client/mac/gcov/libgcov.a",
+        "src/client/mac/handler/breakpad_nlist_64.cc",
+        "src/client/mac/handler/dynamic_images.cc",
+        "src/client/mac/handler/exception_handler.cc",
+        "src/client/mac/handler/minidump_generator.cc",
+        "src/client/mac/handler/obj-cTestCases-Info.plist",
+        "src/client/mac/handler/protected_memory_allocator.cc",
+        "src/common/mac/Breakpad.xcconfig",
+        "src/common/mac/BreakpadDebug.xcconfig",
+        "src/common/mac/BreakpadRelease.xcconfig",
+        "src/common/mac/GTMLogger.m",
+        "src/common/mac/HTTPGetRequest.m",
+        "src/common/mac/HTTPPutRequest.m",
+        "src/common/mac/HTTPRequest.m",
+        "src/common/mac/HTTPSimplePostRequest.m",
+        "src/common/mac/MachIPC.mm",
+        "src/common/mac/arch_utilities.cc",
+        "src/common/mac/bootstrap_compat.cc",
+        "src/common/mac/encoding_util.m",
+        "src/common/mac/file_id.cc",
+        "src/common/mac/launch_reporter.cc",
+        "src/common/mac/macho_id.cc",
+        "src/common/mac/macho_reader.cc",
+        "src/common/mac/macho_utilities.cc",
+        "src/common/mac/macho_walker.cc",
+        "src/common/mac/string_utilities.cc",
+    ]
+
+if env["platform"] == "iphone":
+    breakpad_src += [
+        "src/client/ios/Breakpad.mm",
+        "src/client/ios/Breakpad.xcodeproj/project.pbxproj",
+        "src/client/ios/BreakpadController.mm",
+        "src/client/ios/Breakpad_Prefix.pch",
+        "src/client/ios/exception_handler_no_mach.cc",
+        "src/client/ios/handler/ios_exception_minidump_generator.mm",
+    ]
```

Comment:
`.pbxproj`, `.xcconfig`, `.pch`, `.plist` and `.a` are not source files, and should not be added here.

Probably `src/client/mac/Framework/*.mm` are intended to be built as a separate framework.

Reply:
Removed those files. Also deleted the `.a` file as I couldn't find any reference to it. So probably better to remove it unless it turns out it is really needed on mac.

As I said I don't have a mac to actually build on, but it seems that building breakpad is a bit of a mess on mac. And maybe that separate framework is not even needed at all? https://stackoverflow.com/questions/5404636/building-google-breakpad-on-mac-os-x